zebra-state/src/service/check/tests/utxo.rs

```diff
@@ -53,8 +53,12 @@ fn accept_shielded_mature_coinbase_utxo_spend() {
     };
 
     let result =
-        check::utxo::transparent_coinbase_spend(outpoint, spend_restriction, ordered_utxo.clone());
-    assert_eq!(result, Ok(ordered_utxo));
+        check::utxo::transparent_coinbase_spend(outpoint, spend_restriction, ordered_utxo.as_ref());
+
+    assert!(
+        result.is_ok(),
```

Comment:
Using `assert_eq!` provides better diagnostics on test failure, because it shows the specific error that was returned.

```rust
assert_eq!(result, Ok(()), "mature transparent coinbase spend check should return Ok");
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/6510/commits/e6c6c0f33593095568ffef531379d445f2baa290.